.github/workflows/pr-lint.yaml

```diff
@@ -35,3 +35,14 @@ jobs:
 
       - name: Validate Changeset
         run: pnpm changeset status --since=origin/master
+
+  validate-markdown-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: CI Setup
+        uses: ./.github/actions/ci-setup
+
+      - name: Lint docs
+        run: pnpm lint:docs
```

Comment:
Running `lint:docs` instead of the previous action I was using to temporarily debug something. Will revert back to the action that only scans modified files.